src/components/graph-view.js

```diff
@@ -652,6 +653,20 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       const previousSelection =
         (this.state.selectedNodeObj && this.state.selectedNodeObj.node) || null;
 
+      // Clicking with ctrl will paste
+      if (
+        onPasteSelected &&
+        !readOnly &&
+        d3.event.ctrlKey &&
+        previousSelection
+      ) {
+        const xycoords = d3.mouse(d3.event.target);
+
+        onPasteSelected(xycoords[0], xycoords[1], previousSelection);
```

Comment:
I would suggest using this format:

```
onPasteSelected(previousSelection, xycoords);
```

It follows the order of "what" then "where" to note the subject and the location, instead of "where" and "what".